data/clipper/NOTICE

```diff
@@ -29,3 +29,21 @@ You can obtain a copy of the raw data and its license at:
    https://511.org/developers/list/data/
 
 5. Station, stop and agency ID mappings for Clipper card are not provided by the aforementioned agencies. This information is available in Metrodroid's source repository, and is subject to the same license terms as Metrodroid.
+
+Mapping data is contributed by:
```

Comment:
This should be a list of names like ORCA.  That way we also don't include a bunch of email addresses, too. ;)